docs/Theming.md

```diff
@@ -0,0 +1,49 @@
+# Theming
+
+## Motivation
+We want to give Gardener landscape administrators the possibility to change the appearance of the Gardener Dashboard via configuration without the need to touch the code. Right now, we support basic theming, more theming options would be possible but are currently not planned.
+
+## Colors
+Gardener Dashboard as been built with Vuetify. We use Vuetify's built-in [theming support](https://vuetifyjs.com/en/features/theme/) to centrally configure colors that are used throughout the web application.
+Colors can be configured for both light and dark themes. Configuraion is done via the helm chart, see the respective theme section there. Colors cam be secified as HTM color code (e.g. #FF0000 for red) or by referncing a color from Vuetify's Material Design [Color Pack](https://vuetifyjs.com/en/styles/colors/#javascript-color-pack).
+
+The following colors can colors can be configured:
+
+| name                    | usage|
+| ----------------------- |------|
+| `primary`               | icons, chips, buttons, popovers, etc. |
+| `anchor`                | links |
+| `main-background`       | main navigation, login page   |
+| `main-navigation-title` | text color on main navigation |
+| `toolbar-background`    | background color for toolbars in cards, dialogs, etc. |
+| `toolbar-title`         | text color for toolbars in cards, dialogs, etc. |
+| `action-button`         | buttons in tables and cards, e.g. cluster details page |
+| `info`                  | Snotify info popups |
+| `warning`               | Snotify warning popups, warning texts |
+| `error`                 | Snotify error popups, error texts |
+
+## Logos and Icons
+It is also possible to exchange the Dashboard logo and icons. You can replace the assets folder via a volume mount in your deployment.
```

Comment:
can you be a bit more specific which icons you are referring to?

Reply:
Yes but then we need to keep the docu always up-to-date. WDYT

Reply:
An alternative is to point to the github folder that contains the assets that can be replaced

Reply:
added link to docu